roles/vscode/files/vscode.list

```diff
@@ -0,0 +1 @@
+deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main
```

Comment:
I know they recommend specifying the architecture, but is it really necessary if amd64 is the only one we support anyway?

Reply:
Only because Microsoft provides it that way.